### PR TITLE
AutomationML Vocabulary

### PR DESCRIPTION
**Pull request by igg777**
A vocabulary which represents the AutomationML Standard - IEC 62714

### CODE REVIEW

File: eis/vocabs/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Repository for redirecting to videogame extension vocabularies
```
Comment (davidlehn) [inline eis/vocabs/readme.md]: Copied and forgot to update?

File: eis/vocabs/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+===================
+
+Repository created to store the redirections of the the vocabulary for AutomationML
+
+At the moment we have three games extensions:
```
Comment (davidlehn) [inline eis/vocabs/readme.md]: Forgot to change?

File: eis/vocabs/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+/eis/vocabs/aml: 
+
+Html: https://github.com/i40-Tools/vocabularies/tree/master/AutomationML/amlDocs/
```
Comment (davidlehn) [inline eis/vocabs/readme.md]: Not a comment for this file, but I did notice you were missing some markdown backtick quotes in the readme at this link so the file tree layout diagram there looks wrong.  And I could nitpick that "HTML" here should be all caps. :-)

File: eis/vocabs/readme.md (hunk 2 above)
Comment (igg777) [inline eis/vocabs/readme.md]: Yes. Indeed.

File: eis/vocabs/readme.md (hunk 3 above)
Comment (igg777) [inline eis/vocabs/readme.md]: Ok, thanks :). The link is fine?

Comment (igg777) [inline eis/vocabs/readme.md]: Could you please tell me what I´m doing wrong? I also need to add the rules for Manchester Syntax.

Comment (davidlehn) [inline eis/vocabs/readme.md]: Link is fine if that's what you want.  Wrong?  The "Widico ... 3 folders" block is munged into a paragraph.  You probably want to have triple backticks around it or indent it like it's code so it's formatted properly via markdown.

Hadn't heard of Manchester Syntax until just now.  Probably make a similar rule like the owl and turtle ones above except with MIME type and extension from here:
https://www.w3.org/TR/owl2-manchester-syntax/#Appendix:_Internet_Media_Type.2C_File_Extension_and_Macintosh_File_Type

File: eis/vocabs/.htaccess [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^fuhsen$ https://github.com/LiDaKrA/Ontology/blob/master/Documentation/  [R=303,NE,L]
+RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+RewriteRule ^fuhsen$ https://raw.githubusercontent.com/i40-Tools/vocabularies/master/AutomationML/aml.owl [R=303,NE,L]
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.*
```
Comment (davidlehn) [inline eis/vocabs/.htaccess]: Should likely be "text/owl-manchester".
https://www.w3.org/TR/owl2-manchester-syntax/#Appendix:_Internet_Media_Type.2C_File_Extension_and_Macintosh_File_Type

File: eis/vocabs/readme.md (hunk 3 above)
Comment (igg777) [inline eis/vocabs/readme.md]: Thanks for your support :). I want to publish the vocabulary but every time I try to make the pull request I get an error. I'd like to know why?

Comment (davidlehn) [inline eis/vocabs/readme.md]: Publish how?  You mean the travis-ci errors on this PR?  Ignore those, our testing system is failing on unrelated redirects.

Comment (igg777) [inline eis/vocabs/readme.md]: Yes, what I want to achieve is that when I type this https://w3id.org/eis/vocabs/aml# it load the documentation page of the vocabulary. I tried to follow the steps to do so by no success so far.

File: eis/vocabs/.htaccess (hunk 4 above)
Comment (igg777) [inline eis/vocabs/.htaccess]: I included the Manchester syntax in the redirection rule. Could you please check if this is ok? I really need to have this done for the fushen and aml vocabularies.

**Comment by igg777** [comment]
Could you please merge it again? I need to finish publishing the FuhSen vocabulary

**Comment by davidlehn** [comment]
Again?  Your PR was merged:
https://github.com/perma-id/w3id.org/tree/master/eis/vocabs

**Comment by igg777** [comment]
Yes, but there were problems of configuration. Sorry for my ignorance

**Comment by davidlehn** [comment]
I don't have time to help much with this right now.  You need to test and fix the config as needed.  The text/plain content-type is what github is returning for your aml.owl.  I don't know if or how you can change how that works.

**Comment by igrangel** [comment]
#outlook a {

padding: 0;
}
.ReadMsgBody {

width: 100%;
}
.ExternalClass {

width: 100%;
}
.ExternalClass, .ExternalClass p, .ExternalClass span, .ExternalClass font, .ExternalClass td, .ExternalClass div {

line-height: 100%;
}
body, table, td, a {

-webkit-text-size-adjust: 100%;

-ms-text-size-adjust: 100%;
}
table, td {

mso-table-lspace: 0pt;

mso-table-rspace: 0pt;
}
img {

-ms-interpolation-mode: bicubic;
}

body {

margin: 0;

padding: 0;

}
img {

border: 0;

height: auto;

line-height: 100%;

outline: none;

text-decoration: none;
}
table {

border-collapse: collapse !important;
}
body {

height: 100% !important;

margin: 0;

padding: 0;

width: 100% !important;
}

appleBody a {

color: #68440a;

text-decoration: none;
}
.appleFooter a {

color: #999999;

text-decoration: none;
}
.faq {

line-height: 18px;

list-style: outside none none;

margin: 10px 0;

padding: 0;
}
.mobile-only {

display: none;
}

@media screen and (max-width: 525px) {

table[class="wrapper"] {

width: 100% !important;
}

td[class="logo"] {

text-align: left;

padding: 20px 0 20px 0 !important;
}
td[class="logo"] img {

margin: 0 auto!important;
}

[class="mobile-only"] {

display: block;
}

br[class="mobile-hide"] {

display: none;
}
td[class="mobile-hide"] {

display: none;
}
img[class="mobile-hide"] {

display: none !important;
}
img[class="img-max"] {

max-width: 100% !important;

width: 100% !important;

height: auto !important;
}

img[class="img-min"] {

max-width: 73px !important;

width: 73px !important;

height: auto !important;

padding-top: 8px !important;
}

img[class="img-apps"] {

max-width: 46% !important;

width: 46% !important;

height: auto !important;
}
img[class="img-max-profile"] {

max-width: 50% !important;

width: 50% !important;

height: auto !important;

border-radius: 200px;
}

table[class="responsive-table"] {

width: 100%!important;
}

td[class="padding"] {

padding: 10px 5% 15px 5% !important;
}
td[class="amount"] {

font-size: 10px!important;
}
td[class="mobile-text-twelve"] {

font-size: 12px!important;
}
span[class="mobile-text-twelve"] {

font-size: 14px!important;
}
td[class="padding-copy"] {

padding: 10px 5% 10px 5% !important;

text-align: center;
}
td[class="padding-meta"] {

padding: 30px 5% 0px 5% !important;

text-align: center;
}
td[class="no-pad"] {

padding: 0 0 20px 0 !important;
}
td[class="no-pad-bottom"] {

padding: 25px 0 0 0 !important;
}
td[class="no-padding"] {

padding: 0 !important;
}
td[class="section-padding"] {

padding: 25px 15px 25px 15px !important;
}
td[class="section-no-padding-bottom"] {

padding: 25px 15px 0 15px !important;
}

td[class="section-no-padding-top"] {

padding: 0 5% 25px 5% !important;
}

td[class="section-no-padding-top-bottom"] {

padding: 0 5% 0 5% !important;
}
td[class="section-padding-bottom-image"] {

padding: 50px 15px 0 15px !important;
}

td[class="mobile-wrapper"] {

padding: 10px 5% 15px 5% !important;
}
table[class="mobile-button-container"] {

margin: 0 auto;

width: 100% !important;
}
a[class="mobile-button"] {

width: 80% !important;

padding: 15px !important;

border: 0 !important;

font-size: 16px !important;
}
}

irlangrangel2003@gmail.com has been waiting for you to confirm your relationship:

Confirm &#9658;

This is your final reminder.
Not sure you know irlangrangel2003@gmail.com?

View their photos to see if you recognize them, or simply ignore this email.

*This offer has been sent by irlangrangel2003@gmail.com. By accepting this offer to try the new empowr, I stipulate that I'm 18 or older and am signing up having agreed with the product, terms and privacy policies, including the manner in which "$" and the word "earnings" are used throughout the empowr website, mobile apps and all empowr communications including this email, as defined in the product policy. This promotional email was sent to reply@reply.github.com and may contain special notices or offers. It is for illustrative purposes only and is in no way a guarantee. You can prevent future emails here. empowr - 501 West Broadway, Suite A182, San Diego, CA 92101, USA.

**Comment by igrangel** [comment]
body {{ margin:0 !important; }}
div[style*="margin: 16px 0"] {{ margin:0 !important; font-size:100% !important; }}
#outlook a {{

padding: 0;
}}
.ReadMsgBody {{

width: 100%;
}}
.ExternalClass {{

width: 100%;
}}
.ExternalClass, .ExternalClass p, .ExternalClass span, .ExternalClass font, .ExternalClass td, .ExternalClass div {{

line-height: 100%;
}}
body, table, td, a {{

-webkit-text-size-adjust: 100%;

-ms-text-size-adjust: 100%;
}}
table, td {{

mso-table-lspace: 0pt;

mso-table-rspace: 0pt;
}}
img {{

-ms-interpolation-mode: bicubic;
}}

body {{margin: 0;padding: 0;}}
img {{

border: 0;

height: auto;

line-height: 100%;

outline: none;

text-decoration: none;
}}
table {{

border-collapse: collapse !important;
}}
body {{

height: 100% !important;

margin: 0;

padding: 0;

width: 100% !important;
}}

appleBody a {{

color: #68440a;

text-decoration: none;
}}
.appleFooter a {{

color: #999999;

text-decoration: none;
}}
.faq {{

line-height: 18px;

list-style: outside none none;

margin: 10px 0;

padding: 0;
}}
.mobile-only {{

display: none;
}}
.img-apps {{ padding:0 5px 0 0;}}

a.mobile-button:link, a.mobile-button:visited {{ color:#fff!important;}}

@media screen and (max-width: 525px) {{

table[class="wrapper"] {{

width: 100% !important;
}}

td[class="logo"] {{

text-align: left;

padding: 20px 0 20px 0 !important;
}}
td[class="logo"] img {{

margin: 0 auto!important;
}}

[class="mobile-only"] {{

display: block;
}}

br[class="mobile-hide"] {{

display: none;
}}
td[class="mobile-hide"] {{

display: none;
}}
img[class="mobile-hide"] {{

display: none !important;
}}
img[class="img-max"] {{

max-width: 100% !important;

width: 100% !important;

height: auto !important;
}}

img[class="img-min"] {{

max-width: 73px !important;

width: 73px !important;

height: auto !important;

padding-top: 8px !important;
}}

img[class="img-apps"] {{

max-width: 46% !important;

width: 46% !important;

height: auto !important;

padding:0 2px;
}}
img[class="img-max-profile"] {{

max-width: 50% !important;

width: 50% !important;

height: auto !important;

border-radius: 200px;
}}

table[class="responsive-table"] {{

width: 100%!important;
}}

td[class="padding"] {{

padding: 10px 5% 15px 5% !important;
}}
td[class="amount"] {{

font-size: 9px!important;
}}
td[class="mobile-text-twelve"] {{

font-size: 12px!important;
}}
span[class="mobile-text-twelve"] {{

font-size: 14px!important;
}}
td[class="padding-copy"] {{

padding: 10px 5% 10px 5% !important;

text-align: center;
}}
td[class="padding-meta"] {{

padding: 30px 5% 0px 5% !important;

text-align: center;
}}
td[class="no-pad"] {{

padding: 0 0 20px 0 !important;
}}
td[class="no-pad-bottom"] {{

padding: 25px 0 0 0 !important;
}}
td[class="no-padding"] {{

padding: 0 !important;
}}
td[class="section-padding"] {{

padding: 25px 15px 25px 15px !important;
}}
td[class="section-no-padding-bottom"] {{

padding: 25px 15px 0 15px !important;
}}

td[class="section-no-padding-top"] {{

padding: 0 5% 25px 5% !important;
}}

td[class="section-no-padding-top-bottom"] {{

padding: 0 5% 0 5% !important;
}}
td[class="section-padding-bottom-image"] {{

padding: 50px 15px 0 15px !important;
}}

td[class="mobile-wrapper"] {{

padding: 10px 5% 15px 5% !important;
}}
table[class="mobile-button-container"] {{

margin: 0 auto;

width: 100% !important;
}}
a[class="mobile-button"] {{

width: 80% !important;

padding: 15px !important;

border: 0 !important;

font-size: 16px !important;
}}
}}

Earn $100 in 1 minute.

How?

Simply try our new social network for only 50 seconds...

..and give us your feedback in less than 10 seconds.

Give me $100 (start the clock) &#9658;

Your empowr account was reserved for you by irlangrangel2003@gmail.com. In order to store your gift however you must agree to terms and validate before it can be created.

By accessing your account via the links or buttons in this email, you're both validating your email and stipulating that you're 18 or older and are signing up having agreed with the product, terms and privacy policies, including the manner in which "$", "earnings", maturation and advertising are used or applied throughout the empowr products and communications.

Earnings are virtual currency revenues, not profits; are for illustrative purposes only and are in no way a guarantee of anything.

While your account is very new, if you fail to sign in for just a few days, it will be deleted to eliminate unwanted emails for you, so click a button in emails periodically to notify the system of your desire to keep it active. You can manually delete your account and/or block future invitations from here
http://profile.empowr.com/ApplicationManager/CommunityEmailSettings.aspx?em=K%2FzdUVCDppR7HhgqAnE1e8Nc50J7OfUMz2tTCnD5g7PUM4hU8BkXckO48Mcd9QscBTwTaeHw%2B4PRELE67Irr%2Fgj3HOHkXE%2FHvu9mT5DZoPNhAS4NRyvHvf%2FLdZi4xBgZb4qC20Q6G2gK3QxKlpMiH76CtN%2BrZxB9&vet=724&mlid=9381386265.

*This promotional email was sent to reply@reply.github.com and may contain special notices or offers. To see current terms and conditions visit: empowr Terms and Conditions. You can prevent future emails here. You can change your password here. empowr - 501 West Broadway, Suite A182, San Diego, CA 92101, USA

**Comment by igrangel** [comment]
body { -ms-text-size-adjust: none !important; font-family:'Helvetica Neue', Helvetica, Arial, sans-serif;}

My balance: $1,000

Home

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

&nbsp;

p

small-text

Hello,

Starting in 2016, you were an early (alpha) member of our project called empowr. Well, that project evolved and became the new www.empowr.com. Along the way, we invented virtual currency and many other technologies used by all the top social media companies.

Your free coins

As you probably know by now, empowr is launching its own coin (like Bitcoin) and, as our way of saying Thank You to the incredible community that got us here, we're giving the first coins to you – TOTALLY FREE. Read the details
http://www.empowr.com/Navigation/RegistrationWelcome.aspx?Action=ReferralInstantRegistration&amp;RegistrationFlowFlavor=18&amp;authToken=n3qpiEfuKpusbjq%2bQqPSdx1m%2f7dfL9ernlp5S4lPbB%2fYxQKsDqBGfbRg3WQoXuy318zeG53pzcvVJT9XNdkRr%2f%2b8k3c6t%2byqEGCIG15OUG5MBTUMeQ203yzvN67rHVxOGH5cUUoUP1HLDimeTnqLUPayhru2lgCfU1d1Ul0UTG1j52HBlQzMkOpWzXyqeMSVdt9rPcgSaE0oJ6l5eX%2bX%2blIpg4Kpzl6uBKxeJF9%2b4LJHLF142DR%2fK1ykm2xsoyVfafuD%2b3kF1hFkIQcHqleNEX6uiSM05Rkk%2fhkWCvWvTTsP7P7LLC50usze%2fATiH8nL&amp;PolicyAcceptance=znV6CXDy2VNQ4XnInvtI76aIYgK%2bP0%2bSTnBZLMGIEYkncmWsYye4pv7%2foRtYL4nr9ZbR%2flUmOK9dsdPt4buGl9Pm0vgjapQLqEpo9ErsOPpgiIPlsKrInljFkSoWKrYxlnd7zMT%2bcGhlr1Fl224RKuTCbB2QpqLPl5rPebDoVX1RDj%2fYodenz%2bdk2wXu0sYEY62kcT%2bLotaVDmvTIk7Ed%2bbq1coBlgDc%2ffD291RwOgDbAZvuGpX4JQeM4%2bc76CIft1lO3QGLPoYEp0NnDa1NQVNtYKRQkH7lgiqc2ouv8bOlGYeSe0LuORXNQ4%2bEwzFUxKTNKaaaq3JaiG5iNmjtydWLIdz5KEjPp7aoYkSI9z6HdsGN%2fnlHJazOCxUuYjtCAwG%2f6pocTxpjihMi%2bXZ%2fOpMA%2fOWn9GbaVW26etal5qf3oKNUHptKgoGiZSNWu9gPFIv3sg9k3Lhhr7p9GZGXm3Wc3PbbCMkHwezLtIg5YbZKMJUlc0DJJS7dXaf7US40B%2fsK74xGruGKKqWVBl%2boQLxT8bn7ecHgrOdiw89clUEjgDAUOuJybxZfPzYOUGD4GXfsGIisloGou9N5e7Xmurd62kCJml8Bsu1viE2jqzlmgTmp1VJZnt%2f%2fGNfwEfucehukTpgrbM3rufnebNW5w%2brePA9i8vcDCXR8sL64KRNSQD7w47zhxQi9xmUf6WM97LDxEEju3Z3WnB6w8EH%2fOQ%2bcQD5bHr1BSTl%2bDHhD4K0A9YGMGW5OWWUx7JXbK514LYOOmiSz6sTn4qXWWR35A3Rw%2bVsHoF%2f62Hf32%2fpfCfGVQxSoNnHQwD3sK2MNR77Uu2kPrMmjqI%2bYQPg098Mc6azG7JzbWVOFaKhqxkhUGD9PcFnfewZGoODlf1bicvI21WMfT5bTZx255juDsAf33mqNgc3Kt%2fapb1olxCaDPXOQnAetj81gYknGpZhAp7R5bRYH5vB5VoJsveB8WamE2A%3d%3d&amp;vet=760&amp;dt=011418&amp;mlid=10170036941&amp;tmpID=3378&amp;linkname=InviteLink&amp;pt=0&amp;esp=0&amp;eaid=42921970&amp;rt=1

Secure your free coins &#9658;

Note: Unclaimed coins will be auctioned off to the rest of the community. (As an early member, you get exclusive first access to bid on any unclaimed coins).

Once again, THANK YOU.
-Your friends at empowr.com

Your account was reserved for you in order to allocate your free coins, however you must agree to terms and validate before it can be utilized. By accessing your account via the links or buttons in this email, you're both validating your email and stipulating that you're 18 or older and are signing up having agreed with the product, terms and privacy policies.

At this moment, empowr can make no guarantees about the current or future price or value of the new empowr cryptocurrency coin, or that it will be ultimately tradeable on one or more public currency exchanges. By accepting your free founder coins, you acknowledge that this offer represents a gift that can be revoked if it is found to not be legal in your state, country or jurisdiction. Vesting and other details apply.

You are receiving this gift and email because you were an alpha user of the SMS.ac / FanBox / empowr project. You can unsubscribe here
#.

-->

New! Download the empowr mobile app:

&nbsp;

&nbsp;

*This promotional email was sent to reply@reply.github.com and may contain special notices or offers. To see current terms and conditions visit: FanBox Terms and Conditions. You can prevent future emails here.
FanBox - 611 K Street, Suite B411, San Diego, CA 92101, USA

**Pull request merged**